src/components/message/message-card/header/MessageCardHeader.tsx

```diff
@@ -96,23 +98,27 @@ export const MessageCardHeader = React.memo((props: MessageInfoProps & MessageCa
 				{isBookmarked && <div className={bookmarkIconClass} />}
 				{isAttached && <div className='mc-header__attached-icon' />}
 			</Chip>
-			<Chip
-				className={timestampClassName}
-				title={`Timestamp: ${formattedTimestamp}`}
-				onMouseEnter={onTimestampMouseEnter}
-				onMouseLeave={onTimestampMouseLeave}>
-				{timestamp && formattedTimestamp}
-			</Chip>
+			{!isCollapsed && (
```

Comment:
Instead of boolean can we have an enum just as we use in EventCardHeader?
```
const enum CardDisplayType {
	STATUS_ONLY = 'status-only',
	MINIMAL = 'minimal',
	FULL = 'full',
}
```